### PR TITLE
feat(rvm): add `rb31` shortcut to use ruby 3.1

### PR DESCRIPTION
**Pull request by cIvanrc**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- Added ruby3.1.1 to rvm plugin

## Other comments:

...

### CODE REVIEW

**Review by carlosala — APPROVED** [review]

File: plugins/rvm/rvm.plugin.zsh [hunk 1]
```diff
@@ -26,6 +26,7 @@ rubies=(
   26  'ruby-2.6'
   27  'ruby-2.7'
   30  'ruby-3.0'
+  31  'ruby-3.1.1'
```
Comment (mcornella) [inline plugins/rvm/rvm.plugin.zsh:29]: Shouldn't this be `ruby-3.1` in case a new patch version comes out?

**Review by carlosala — APPROVED** [review]
Now it looks ready to merge! Thanks for the contribution Ivan! 🎉

**Review by Tjolk072 — COMMENTED** [review]
Thanxxxxxxxxxxxxxxxxx

**Review by Tjolk072 — COMMENTED** [review]
You're the best, thanks

**Pull request merged**